File: musicRxJava/src/main/java/com/svalero/musicrxjava/domain/Artist.java
package com.svalero.musicrxjava.domain;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor

public class Artist {

    private long id;

    private String name;

    private String registrationDate;

    private boolean isSoloist;

    private String image;

    private String country;
}

File: musicPhp/models/Artist.php
<?php

namespace models;

class Artist {

    public $id;
    public $name;
    public $country;
    public $registrationDate;
    public $isSoloist;
    public $image;



    public function __construct($data) {
        $this->id = $data["id"];
        $this->name = $data["name"];
        $this->country = $data["country"];
        $this->registrationDate = $data["registrationDate"];
        $this->isSoloist = $data["soloist"];
        $this->image = $data["image"];
    }

}

?>